extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.css

```diff
@@ -31,7 +31,7 @@
  */
 .amp-lightboxed {
   position: fixed !important;
-  z-index: 2147483645 !important;
+  z-index: 2147483644 !important;
```

Comment:
fixing the sidebar z-index should be simple. I dont think we need a new issue to fix this.